_glossary/amortisation.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Amortisation
+---
+
+When discussing [Embodied Carbon](#embodied-carbon), amortisation refers to dividing the total embodied carbon of a device by its expected lifespan. This gives a way of accounting for the emissions a device on a yearly basis. For example; A [2023 Apple 16" Macbook Pro has an embodied carbon of 290kg CO2e](https://www.apple.com/environment/pdf/products/notebooks/16-inch_MacBook_Pro_PER_Oct2023.pdf) with a life cycle of four years gives an amortisation of:
```

Comment:
Colon after "example", and lower-case 'A' immediately afterwards
After CO2e use "so with ... that gives", or "which with ..."